src/components/Cards.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+
+import '../styles/components/Cards.scss';
+
+class Cards extends React.Component{
+  constructor(){
+    super();
+
+    this.state = {
+      currentCardIndex: 0
+    }
+
+
+  }
+
+
+render(){
+  return(
+  <div>
+    <div className="card">
+      {this.props.playerCard.name}<br></br>
+      <img className="sprite" src={this.props.playerCard.sprites.front_shiny} /><br></br>
+      Speed: {this.props.playerCard.stats[0].base_stat}<button stat={this.props.playerCard.stats[0].base_stat} name="speed" onClick={ event =>  this.props.handleClick(this.props.playerCard.stats[0].base_stat, this.props.computerCard.stats[0].base_stat) }>Select</button><br></br>
```

Comment:
I wonder if this might better generated using a loop rather than code duplication. The only difference between the rows is the label and index. You could create an array of labels, map over them and generate each button using the label from array and its index